src/components/LayoutTree.tsx

```diff
@@ -665,6 +680,12 @@ const LayoutTree = () => {
             // Perform this check here instead of in virtualThoughtsPositioned since it changes with the scroll position (though currently `sizes` will change as new thoughts are rendered, causing virtualThoughtsPositioned to re-render anyway).
             if (belowCursor && !isCursor && y > viewportBottom + height) return null
 
+            // Get the next and previous thoughts
```

Comment:
This comment does not seem to match the code. I think it's just getting the previous thought, not the next and previous thoughts.

Reply:
Yes, I forgot to update it, I was getting the `next` as well there when I wrote that comment before removing the `next`.